src/command-line/start.js

```diff
@@ -15,26 +15,29 @@ program
 	.option("    --private", "start in private mode")
 	.description("Start the server")
 	.action(function(options) {
-		var users = new ClientManager().getUsers();
+		let manager = new ClientManager();
 
-		var mode = Helper.config.public;
-		if (options.public) {
-			mode = true;
-		} else if (options.private) {
-			mode = false;
-		}
+		manager.getUsersPromise()
+			.then((data) => {
+				let users = data.users;
+				let mode = Helper.config.public;
+				if (options.public) {
+					mode = true;
+				} else if (options.private) {
+					mode = false;
+				}
 
-		if (!mode && !users.length && !Helper.config.ldap.enable) {
-			log.warn("No users found.");
-			log.info(`Create a new user with ${colors.bold("lounge add <name>")}.`);
+				if (!mode && !users.length && !Helper.config.ldap.enable) {
```

Comment:
Can you move this check to `autoloadUsers`? Then `start` CLI command won't need to deal with users at all.